nixos/modules/services/networking/xrdp.nix

```diff
@@ -109,13 +334,28 @@ in
 
     fonts.enableDefaultFonts = mkDefault true;
 
+    environment.etc = {
+      "xrdp/xrdp.ini".text = xrdpIni;
+      "xrdp/sesman.ini".text = sesmanIni;
+      "xrdp/startwm.sh".source = startwm;
+
+      "xrdp/xrdp_keyboard.ini".source = "${cfg.package}/etc/xrdp/xrdp_keyboard.ini";
+      "xrdp/pulse/default.pa".source = "${cfg.package}/etc/xrdp/pulse/default.pa";
+    } // (mapAttrs (k: _: {
+      target = "xrdp/${k}";
+      source = "${pkgs.xrdp}/etc/xrdp/${k}";
+    }) kmFiles);
+
     systemd = {
       services.xrdp = {
         wantedBy = [ "multi-user.target" ];
-        after = [ "network.target" ];
+        after = [ "network.target" "xrdp-sesman.service" ];
         description = "xrdp daemon";
-        requires = [ "xrdp-sesman.service" ];
         preStart = ''
+          # create run directory
+          mkdir -p /run/xrdp
```

Comment:
```
serviceConfig.RuntimeDirectory = "xrdp";
```

Reply:
@Mic92, this PR is based on #83927 and #83922 so includes them as first commits. Your comments should belong to #83927 and #83922